src/cargo/ops/cargo_test.rs

```diff
@@ -215,6 +217,14 @@ fn run_doc_tests(
         for (var, value) in env {
             p.env(var, value);
         }
+
+        let color_arg = match color {
+            ColorChoice::Always => "always",
+            ColorChoice::Never => "never",
+            ColorChoice::CargoAuto => "auto",
+        };
+        p.arg("--color").arg(color_arg);
```

Comment:
Just want to call out that this may break others' builds if they already have `--color` in their `RUSTDOCFLAGS`. `rustdoc` doesn't allow more then one occurrence of the `--color` flag. While we [don't consider this a breaking change](https://doc.rust-lang.org/nightly/cargo/reference/config.html#buildrustdocflags), this is worth mentioning in changelog.